packages/react-core/src/components/Timestamp/examples/Timestamp.md

```diff
@@ -0,0 +1,59 @@
+---
+id: Timestamp
+section: components
+cssPrefix: pf-c-timestamp
+propComponents: ['Timestamp']
+beta: true
+---
+
+## Examples
+
+### Default
+
+By default, a timestamp will display the current date and time based on the current locale if the `date` prop is not passed in.
+
+The following example shows one way to build an ISO 8601 string with the current local date and time, which gets passed into the `datetime` prop. Alternatively, the current UTC date and time could be passed in using the `toISOString` method.
+
+```ts file="./TimestampDefault.tsx"
+```
+
+### Basic formats
+
+The format of the displayed content can be customized by passing in the `dateFormat` and/or `timeFormat` props. Passing in only one of the props will display only the date or time, depending on which prop is passed in. The possible options are "full", "long", "medium", and "short".
```

Comment:
> Looks like we're missing the __text class on the time element. I'm assuming because that isn't showing up in react-styles since the class doesn't have any styles from core? In those instances, I believe what we do is hardcode the class so that it's there and we can reference it if we need without introducing a new class, but maybe one of the react devs can verify if that's still the case.

That was my assumption as well when I saw that class not able to be applied via `styles...`. For now I hardcoded the class in.

> There is some extra whitespace being created within the <time> element - at both the start and end of the text with custom content, and at the end of the regular examples...

Good catch! I've updated locally how the default content gets rendered so the trailing whitespace at the end should not longer be there. In terms of what it looks like in dev tools, I noticed that the Title component examples look similar (with a white space between the `>` and `<` brackets). It doesn't look like that is affecting what is actually rendered at least now.

With the timeZoneSuffix:

![timestamp with timezone suffix](https://user-images.githubusercontent.com/70952936/185397771-b6f7cd37-54f1-4c67-babd-c6559b6b44ac.png)

Without the suffix:

![timestamp without timezone suffix](https://user-images.githubusercontent.com/70952936/185397552-7ad4712d-2eaa-4195-b885-ff1dc5b0f736.png)

For the outline, I agree it does look a bit tight. This is what it looks like with 0.25rem (I believe the global xs spacer CSS variable):

![timestamp outline](https://user-images.githubusercontent.com/70952936/185399089-2981c51b-6b7f-42e6-89a4-2c1ab9663e00.png)

I also had tried just using 2px and it looked okay as well, but the dashed underline gets cut off with that. If we want to add an outline offset I can open up a PR in Core to add that in.

Reply:
@thatblindgeye go ahead and open a PR, that would be great - thanks! What we typically do in core is use an SCSS function to render the correct `rem` value.

> I noticed that the Title component examples look similar (with a white space between the > and < brackets). It doesn't look like that is affecting what is actually rendered at least now.

Yeah, I imagine that's a common thing. It isn't really visible and _probably_ not a thing unless there is some kind of indication of where the text/thing (like an image or icon) in the thing stops and starts - the reason I noticed it is because the text has the underline. The title component is a block element so it wouldn't have that issue, since it will push everything before/after it onto different lines. Also there isn't an indication of where it stops/starts unless we added a border or background or something.

A really common problem I notice all over the place is with links. It's visible in our text component, too:

<img width="1742" alt="Screen Shot 2022-08-18 at 10 38 22 AM" src="https://user-images.githubusercontent.com/35148959/185456656-09b37ca8-a081-4a7c-952c-f5eebe194673.png">


<img width="1742" alt="Screen Shot 2022-08-18 at 10 38 34 AM" src="https://user-images.githubusercontent.com/35148959/185456523-401d5010-2156-43f0-8cb9-cfd0911e0847.png">